database/buildSchema/22_action_plugin.sql

```diff
@@ -1,12 +1,14 @@
 -- action plugins
 
 CREATE TABLE public.action_plugin (
---    id serial primary key,
     code varchar primary key,
     name varchar,
+    type varchar,
     description varchar,
     path varchar,
+    file varchar,
```

Comment:
Why do we need the `file` field? The `path` field already contains the filename in it (it created using `folder + path` when loaded), so we're just storing the same thing twice.

It should either be `folder_path` and `file` which we combine at run-time, or just `path` which contains the full path. I'd opt for the latter, since the full path is the actual relevant info and only ever used in its joined state.

Reply:
I agree. Again, coming from the `plugin.json`